src/generated/type/src/model.rs

```diff
@@ -763,11 +805,17 @@ impl Money {
 /// For instance, in Java this would be:
 ///
 ///    com.google.type.PhoneNumber wireProto =
-///        com.google.type.PhoneNumber.newBuilder().build();
+/// ```norust
```

Comment:
I think my hack is not doing well in this case.  Blockquotes require a blank line before they start and my hack does not require this. The code upstream is only indented 3 spaces, so about 1/2 of the lines appear to be blockquotes.

I am going to let it stand.  The true fix is to use a Markdown parser, as suggested in #92.